### PR TITLE
 app.js: Add coala association

### LINKED ISSUE #17: Add coala association to hemang

### PR DESCRIPTION
**Pull request by hemangsk**
Closes #17

### CODE REVIEW

**Comment by sils** [comment]
see https://github.com/viper-development/viper-development.github.io/issues/24 you're destroying my little concept of the one-short-description-per-person :P

**Comment by hemangsk** [comment]
Ready for review 😄

File: assets/app.js [hunk 1]
```diff
@@ -4,40 +4,46 @@ app.controller('Data', ['$http', function($http , $scope){
 	self.profile = [
 		{
 			"name" : "Lasse Schuirmann",
-			"id" : "@sils",
+			"id" : "<a href='http://github.com/sils'>@sils</a>",
```
Comment (sils) [inline assets/app.js]: nah your JS should generate this link

File: assets/app.js [hunk 2]
```diff
@@ -4,40 +4,46 @@ app.controller('Data', ['$http', function($http , $scope){
...
+			"short_tagline": "",
 			"bio": "Community evangelist. Leader of the <a href='http://coala.io'>coala</a> community team and creator of the LineSpots algorithm."
 		},
 		
                 {
                         "name" : "Sebastian Latacz",
-                        "id" : "@seblat",
+                        "id" : "<a href='http://github.com/seblat'>@seblat</a>",
                         "picture": "https://avatars0.githubusercontent.com/u/18051901",
+                        "short_tagline":"",
```
Comment (sils) [inline assets/app.js]: that'd be `Business Expert`, same for the others, just take the first thing it says as tagline

File: team.md [hunk 3]
```diff
@@ -25,9 +25,9 @@ order: 4
                 </center>
                 <br>
                 <div class="person-name">{{mem.name}}</div>
-                <div class="person-github-id">{{mem.id}}</div>
+                 <div class="person-short-tagline">{{mem.short_tagline}}</div>
```
Comment (sils) [inline team.md]: indentation mismatch

**Comment by hemangsk** [comment]
Its ready for review :tada:

**Comment by sils** [comment]
ack https://github.com/viper-development/viper-development.github.io/pull/22/commits/3c3a1f4d65673eab4d4270f974acd78647ab8577

**Review by sils — APPROVED** [review]

**Pull request merged**